lib/cmd/common/build.js

```diff
@@ -46,31 +31,26 @@ function build(argv, cb) {
   try {
     var argObj = common.parseArgs(args);
     validateArgs(argObj);
-    if (isNGUI()) {
-      validateArgsNgui(argObj);
-    }
+    validateArgsNgui(argObj);
     mutateGitRefArgs(argObj);
     doBuild(argObj, cb);
   } catch (x) {
     log.silly(x);
-    return cb(i18n._("Error processing args: ") + x + "\n" + i18n._("Usage: ") + isNGUI ? build.usage_ngui : build.usage);
+    return cb(i18n._("Error processing args: ") + x + "\n" + i18n._("Usage: ") + build.usage);
   }
 }
 
 function buildForIos(args) {
   return args.destination === "iphone" || args.destination === "ipad" || args.destination === "ios";
 }
 
-function isNGUI() {
-  return ini.get('fhversion') >= 3;
-}
 
 // runs through the args we got and validates as per our build rules
 // TODO - may need to be more specific depending on build target type
 function validateArgs(args) {
   //looks for set alias
   if (args.app) { /*look for alias*/
-    args.app = fhc.appId(args.app);
+    args.app = args.app;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove or correct this useless self-assignment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1656)